doc/ext-pake/api/pake.rst

```diff
@@ -1084,8 +1088,14 @@ Multi-part PAKE operations
         The attributes for the new key.
         This function uses the attributes as follows:
 
-        *   The key type is required. It must be an unstructured key type that can be constructed directly from the PAKE shared secret. For example, :code:`PSA_KEY_TYPE_DERIVE`, :code:`PSA_KEY_TYPE_HMAC`, or :code:`PSA_KEY_TYPE_AES`.
-        *   The key size in ``attributes`` must be zero. The returned key size is always determined from the PAKE shared secret.
+        *   The key type is required.
+            All PAKE algorithms can output a key of type :code:`PSA_KEY_TYPE_DERIVE` or :code:`PSA_KEY_TYPE_HMAC`.
+            PAKE algorithms that produce a pseudo-random shared secret, can also output block-cipher key types, for example :code:`PSA_KEY_TYPE_AES`.
+
```

Comment:
Is this empty line intentional?

Reply:
In this case, I think not, it was probably an artefact of cut and pasting the text below the blank line.